app/outliers.py

```diff
@@ -231,7 +156,7 @@ def print_analysis_summary(analyzed_models):
         errored_models = [analyzer for analyzer in analyzed_models if analyzer.unknown_error_analysis]
 
         # Check the result of the analysis
-        if len(errored_models) > 0:
+        if errored_models:
```

Comment:
Same that `files_changed`, here `errored_models` is a list. So we based on the fact that an empty list is `False`